README.md

```diff
@@ -1,3 +1,38 @@
-# Exploring the set of interoperable features in the Web Platform.
+# [`feature-set`](https://github.com/web-platform-dx/feature-set): exploring the set of interoperable features of the web platform
 
-See the [2022 backgrounder](2022-backgrounder.md) for details.
\ No newline at end of file
+By the W3C WebDX Community Group and contributors
+
```

Comment:
Is this line break intentional?

Reply:
Yes, it looks slightly less cryptic, rendered as HTML (as in https://github.com/ddbeck/feature-set/blob/ad439fc24e0d3acc3321994edd2664c73c16373c/README.md).

Reply:
👍